src/FsAutoComplete.Core/WorkspacePeek.fs

```diff
@@ -110,6 +110,7 @@ let private partitionByChoice3 =
     Array.fold foldBy ([],[],[])
 
 let peek (rootDir: string) deep (excludedDirs: string list) =
+    if isNull rootDir then [] else
```

Comment:
This happens when you have not opened a directory in Code. I test the following scenario:
- Open Code
- Open Fake Script file

Expected:
- Everything works

This nullcheck ensures no exceptions are thrown (which make debugging easier in the above scenario)
I will send a similar change to the ionide extension as there is no need to even make the request in that scenario.